x-pack/plugins/ml/public/maps/anomaly_source.tsx

```diff
@@ -44,6 +48,8 @@ export interface AnomalySourceDescriptor extends AbstractSourceDescriptor {
 export class AnomalySource implements IVectorSource {
   static mlResultsService: MlApiServices['results'];
   static canGetJobs: boolean;
+  static mlLocator?: LocatorPublic<SerializableRecord>;
```

Comment:
Having these optional static variables doesn't seem right to me.
Can these be added to the constructor? or if not, should they be extracted to a new class? 

Also. it looks like `canGetJobs` is never used.